app/views/books/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if books.any? %>
+  <ul class="books">
+    <% books.each do |book| %>
+      <li><%= book.title %></li>
+    <% end %>
+  </ul>
+<% else %>
+  You haven't added any books yet.
```

Comment:
Particularly I only use Internationalization when this makes sense for the project requirements. But sounds like a good approach to follow the Thoughtbot good practices.